.idea/misc.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
you need to remove this file and all files under .idea folder from vcs. Try reading this page(https://stackoverflow.com/questions/36163781/remove-file-from-version-control-in-intellij-idea) or find a solution by yourself

Reply:
Thanks, done.
Shall I merge pull request now after you approve?

Reply:
Yes, please. Good job